internal/tcpchecker/portchecker.go

```diff
@@ -0,0 +1,27 @@
+package tcpchecker
+
+import (
+	"time"
+
+	"github.com/lunarway/strong-duckling/internal/daemon"
+	"github.com/prometheus/common/log"
+)
+
+type PortChecker struct {
+	address string
+	port    int
+}
+
+func StartPortChecking(address string, port int, reporter Reporter) *daemon.Daemon {
+	logger := log.With("name", "portchecker").With("address", address).With("port", port)
+	logger.Infof("Start checking address %s:%v", address, port)
+	daemon := daemon.New(daemon.Configuration{
+		Logger:   logger,
+		Interval: 1 * time.Second,
+		Tick: func() {
+			str, err := CheckPort(address, int(port), reporter)
+			logger.Debugf("Output: %s. Err %s", str, err)
```

Comment:
Should this maybe be split into an output debug log and an error `err` log?

```suggestion
			if err != nil {
				logger.Errorf("tcpchecker on %s:%d failed: %+v", address, port, err)
			}
			logger.Debugf("tcpchecker output for  %s:%d: %s", address, port, str)
```